doc/sphinx/source/community/detailed_release_procedure.rst

```diff
@@ -0,0 +1,190 @@
+and deployed on conda-forge and PyPI; it also assumes the release manager has access to accounts on DKRZ/Levante.
+
+Below is a list of steps that the release manager, together with the previous release manager, should go through before the actual release;
+these include testing the new code by running all available recipes in yhe `main` branch, and comparing the output against
+the previous release.
+
+Open an issue on GitHub
+-----------------------
+
+First, open an issue on GitHub where the testing workflow before the release is documented (see example https://github.com/ESMValGroup/ESMValTool/issues/2881).
+Name it something relevant like "Recipe testing and comparison for release 2.x.x", and populate the issue description with information
+about where the testing is taking place, what tools are used, and what versions, here are some suggestions:
+
+
+Where the work is done e.g. work done on DKRZ/Levante:
+
+- submit files in `/home/b/b382109/submit` (for v2.7.0)
+- output in `/scratch/b/b382109/esmvaltool_output` (for v2.7.0)
+
+We should document various utilities' versions so that the work can be reproduced in case there
+is an issue, or release work needs to be picked up mid-release by another release manager:
+
+- documenting `conda`/`mamba` versions:
+
+.. code-block:: bash
+
+  mamba --version
+
+- documenting `git` branch and its state:
+
+.. code-block:: bash
+
+  git status
+
+Furthermore, the runtime environment needs to be documented:
+
+.. code-block:: bash
+
+  mamba env create -n tool2xxTest -f environment.yml
+  conda activate tool2xxTest
+
+Make a copy of the environment file, and attach it in the release testing issue; to
+record the environment in a yaml file use e.g. `conda env export > ToolEnv270Test.txt`.
+
+Mention any special data movement one needs to perfom to run the tests e.g.:
+for v2.7.0 the release manager moved the autoassess-specific files to
+`/home/b/$USER/autoassess_files` on DKRZ/Levante.
+
+Modifications to configuration files need to be documented as well: for example,
+for v2.7.0 the release manager had to modify `config-user.yml` file by
+adding a number of extra paths for DKRZ/Levante-specific data pools:
+
+.. code-block:: yaml
+
+  CMIP6:
+    - /work/bd0854/DATA/ESMValTool2/CMIP6_DKRZ
+    - /work/bd0854/DATA/ESMValTool2/download/CMIP6
+  CMIP5:
+    - /work/bd0854/DATA/ESMValTool2/CMIP5_DKRZ
+    - /work/bd0854/b309141/additional_CMIP5
+    - /work/bd0854/DATA/ESMValTool2/download/cmip5/output1
+    - /work/bd0854/DATA/ESMValTool2/download/cmip5
+
+Submit run scripts - test recipe runs
+-------------------------------------
+
+We are now ready to start running all the available recipes, to compare output against previous release. Running is currently done
+via batch scripts submitted to a schedulers (SLURM). Generate the submission scripts using the `generate.py` Python script;
+you can find a copy of the script either in `/home/b/b382109/Tool_Release_270_Scripts` or
```

Comment:
I would recommend against adding people's home directories to our public documentation. This is likely to become outdated quickly and maybe you don't need everyone and their dog to start having a look at that.

Reply:
I have just created a directory in `work` named `/work/bd0854/esmvaltool_releases` to store these kinds of files. Would it be an option to place the submission scripts and other scripts inside that common directory instead of people's home directories? Feel free to add subdirectories in it with explicit names, e.g. `sbatch_scripts`, `output_parsing_scripts`, ...

Reply:
yes I agree and thanks, Remi - you mind adding that info here pls?

Reply:
Sure, will do that when I get to these steps during the release. I'll have a better idea of how things could be organized in a more sustainable way

Reply:
I don't want to sound harsh, but code should really be stored in a version controlled repository, not in some directory on some computer. If you do not feel comfortable having it on the ESMValTool repository, we could create a new repository for testing scripts.

Reply:
@bouweandela, you're fully right. These scripts can be found in the draft PR #2883. Before deciding something about these scripts, we first need to agree on a common approach to run all recipe for the testing rounds. This will be discussed next Tuesday at the Tech Lead Team Meeting, see https://github.com/ESMValGroup/Community/discussions/79#discussioncomment-5086745. 

I personally think that these scripts are our best option for the v2.8 release and the near future. These have proven to be very useful for the past 2 releases. If we all agree on that, I'm happy to take a look at #2883, update it if needed and merge it before the release to have the scripts in v2.8. We would then update the documentation for the release procedure accordingly. Once a more advanced testing framework like the recipe test workflow is published, we could think of retiring these scripts.

Reply:
See also the comment here: https://github.com/ESMValGroup/ESMValTool/pull/3032#discussion_r1116639801